src/components/AppProvider/utilities/createAppProviderContext/createAppProviderContext.ts

```diff
@@ -1,75 +0,0 @@
-import createApp, {
```

Comment:
This was added back because of a content change, that I added to `createAppBridge` in `src/utilities/app-bridge`